etc/glite-info-update-endpoints.conf

```diff
@@ -6,3 +6,9 @@ manual = False
 manual_file = 
 output_file = /var/cache/glite/top-urls.conf 
 cache_dir = /var/cache/glite/glite-info-update-endpoints
+# Enable certificate verification on CentOS7 while downloading site
```

Comment:
great-why can't we have the endpoints here too ?

Reply:
OK ! Would it be too much to ask to update this PR with them in ? @gwarf what do you think?